src/sagemaker/session.py

```diff
@@ -495,7 +495,7 @@ def stop_tuning_job(self, name):
                 raise
 
     def transform(self, job_name, model_name, strategy, max_concurrent_transforms, max_payload, env,
-                  input_config, output_config, resource_config, tags):
+                  input_config, output_config, resource_config, data_processing, tags):
```

Comment:
usually we try and put the new argument at the end of the list so that it's not a breaking change if someone had been calling with function without labeling the args before

Reply:
I moved it to end. also fixed some UTs